src/styles/shared/_index.scss

```diff
@@ -0,0 +1,2 @@
+@use './components/Buttons' as buttons;
+@include buttons.styles;
```

Comment:
Note that the SCSS entry point for shared `Buttons` has a _mixin_ to actually output its styles (classname rules). This is for two reasons:

* Allows configuration of the module before the styles are generated
* Allows use of the module without outputting the styles (say, for extending button classes)